cmd/pump/pump.toml

```diff
@@ -30,6 +30,12 @@ pd-urls = "http://127.0.0.1:2379"
 # [storage]
 # Set to `true` (default) for best reliability, which prevents data loss when there is a power failure.
 # sync-log = true
+
+# stop write when disk available space less then the configured size
+# 42 MB -> 42000000, 42 mib -> 44040192
```

Comment:
`one gib` is too small,  use 10% or 10 gib is better

Reply:
it may be annoying to set such a limit in a test or development env, how about default 0 and set
10 gib if deployed by ansible ?

Reply:
if so, use 10% is better, right?

Reply:
update to be 10 gib in https://github.com/pingcap/tidb-binlog/pull/647/commits/ac51fc0d36c528f1b232887bd9f04700792ee0b2